src/components/va-alert/va-alert.tsx

```diff
@@ -0,0 +1,117 @@
+import { Component, Event, EventEmitter, Host, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'va-alert',
+  styleUrl: 'va-alert.css',
+  shadow: true,
+})
+export class VaAlert {
+  /* Determines the icon and border/background color.
+   * One of `info`, `error`, `success`, `warning`, or `continue`
+   */
+  @Prop() status: string = 'info';
+
+  /**
+   * If true, renders the alert with only a background color corresponding
+   * to the status - no icon or left border.
+   */
+  @Prop() backgroundOnly: boolean = false;
+
+  /**
+   * If true, doesn't fire the CustomEvent which can be used for analytics tracking
+   */
+  @Prop() disableAnalytics: boolean = false;
+
+  @Prop() visible: boolean = true;
+
+  /**
+   * Headline for the alert. Won't show if `backgroundOnly` is true
+   */
+  @Prop() headline: string;
+
+  /**
+   * Determines the level of the heading in the alert
+   */
+  @Prop() level: number = 3;
+
+  @Prop() closeBtnAriaLabel: string = 'Close notification';
+
+  @Prop() closeable: boolean = false;
+
+  @Event({
+    composed: true,
+    bubbles: true,
+  })
+  close: EventEmitter;
+
+  @Event({
+    eventName: 'component-library-analytics',
+    composed: true,
+    bubbles: true,
+  })
+  componentLibraryAnalytics: EventEmitter;
+
+  private closeHandler(e: MouseEvent): void {
+    this.close.emit(e);
+  }
+
+  private handleAlertBodyClick(e: MouseEvent): void {
+    if (!this.disableAnalytics) {
+      const target = e.target as HTMLElement;
+      // If it's a link being clicked, dispatch an analytics event
+      if (target?.tagName === 'A') {
+        const detail = {
+          componentName: 'AlertBox',
+          action: 'linkClick',
+          details: {
+            clickLabel: target.innerText,
+            headline: this.headline,
+            status: this.status,
+            backgroundOnly: this.backgroundOnly,
+            closeable: this.closeable,
+          },
+        };
+        this.componentLibraryAnalytics.emit(detail);
+      }
+    }
+  }
+
+  render() {
+    const {
+      backgroundOnly,
+      headline,
+      level,
+      status,
+      visible,
+      closeable,
+    } = this;
+    const classes = `alert ${status} ${backgroundOnly ? 'bg-only' : ''}`;
+
+    if (!visible) return <div aria-live="polite" />;
+
+    return (
+      <Host>
+        <div class={classes}>
+          <i aria-hidden="true" role="img"></i>
+          <span class="sr-only">Alert: </span>
```

Comment:
@1Copenut How will this affect screen readers? Specifically, if this `span.sr-only` comes before the heading and the screen reader user is navigating by heading.

And to be clear, while I definitely want to make sure we do the right thing, if we don't know what that is yet, I don't want it to block this PR. Adding this bit is an improvement over [what we've currently got](https://github.com/department-of-veterans-affairs/component-library/blob/master/src/components/AlertBox/AlertBox.jsx#L101-L114). (Unless, of course, you think it's actually detrimental, in which case we'll just remove it until we come up with the right answer.)

Reply:
Just adding that this was requested by @joshkimux 

https://github.com/department-of-veterans-affairs/va.gov-team/issues/19674

Reply:
Ahh, I gotchu. :+1: